### PR TITLE
Add a list of nodes and reason why they can add latency, in an inform…

### PR DESCRIPTION
**Pull request by padenot**
…ative section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7706,6 +7706,37 @@ <h2 id="Performance">
...
+          <li>The <a>AudioWorker</a> can run a script that buffers internaly,
+          adding delay to the signal path
+          </li>
+          <li>The <a>DelayNode</a>, whose role is to add controlled latency
+          time.
+          </li>
+          <li>The <a>BiquadFilterNode</a> and <a>IIRFilterNode</a> filter
+          design can delay incoming samples
+          </li>
+          <li>The <a>ConvolverNode</a> depending on the impulse, can delay
```
Comment (billhofmann) [inline index.html]: grammar - comma before " depending"

File: index.html [hunk 2]
```diff
@@ -7706,6 +7706,37 @@ <h2 id="Performance">
...
+          incoming samples
+          </li>
+          <li>The <a>DynamicsCompressorNode</a> has a look ahead algorithm that
+          causes delay in the signal path
+          </li>
+          <li>The <a>MediaStreamAudioSourceNode</a> and
+          <a>MediaStreamAudioDestinationNode</a>, depending on the
+          implementation, can add buffer internaly and add delays
```
Comment (billhofmann) [inline index.html]: sp: internally, I expect this is caught elsewhere

**Comment by padenot** [comment]
Rebased, squashed and tidied.

**Comment by padenot** [comment]
Ping ?

**Comment by rtoy** [comment]
lgtm.

**Pull request merged**